crates/pathfinder/src/core.rs

```diff
@@ -41,6 +41,25 @@ pub struct ContractCode {
     pub abi: String,
 }
 
+/// A Starknet contract's definition.
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
+pub struct ContractDefinition {
+    pub abi: String,
+    pub program: contract_definition::Program,
```

Comment:
Is there a reason this isn't just `bytecode: Vec<StarkHash>`? Are we going to be expanding this `Program` struct?

Reply:
If we want to flatten the structure, then `contract_definition` should return `ContractCode`, and this type should be removed. Otherwise we can leave it, and whether the structure should be flattened or closely follow `sequencer::reply::ContractDefinition`, I have no preference of.

Reply:
Structure flattened.